src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigCol {
+    private double side;
+
+    public Square(double side, Color color) {
```

Comment:
Don't pass expressions in the constructor. For better code readability, create separate variables and pass them on. For example, if you're generating a random side length, do it before the constructor call and pass the variable.